nmostesting/suites/BCP0060102Test.py

```diff
@@ -913,3 +913,134 @@ def test_04(self, test):
 
         except TestingFacadeException as e:
             return test.UNCLEAR(e.args[0])
+
+    def test_05(self, test):
```

Comment:
Presumably a significant overlap with `IS0503Test.test_02` but too soon to try to factor out a utility function?

https://github.com/AMWA-TV/nmos-testing/blob/28eeadcd058aa17b6e01ab625b2dd0cdd258585b/nmostesting/suites/IS0503Test.py#L126

Reply:
I've simplified, but didn't use a utility function as the way the SDP was being generated was slightly different to how it is in the mock Node, as here *all* template files are opened at start of test.